### ISSUE TITLE
"unrolling still effective" sample code not working

### ISSUE THREAD

**Issue by doggy8088** [dgryski/go-perfbook#59]
https://play.golang.org/p/6tnySwNxG6O

![image](https://user-images.githubusercontent.com/88981/88245752-ec693780-ccca-11ea-8758-6a9f10d026ab.png)

**Comment by dgryski** [comment]
I don't think this code was meant to be a complete runnable code sample.  Leaving this open until I fix the wording to make this clearer or to make a runnable benchmark.
